modules/items/item-sheet-ffg.js

```diff
@@ -111,9 +111,13 @@ export class ItemSheetFFG extends ItemSheet {
 
         if (!this.item.flags?.starwarsffg?.loaded) {
           CONFIG.logger.debug(`Running Item initial load`);
+          if (!Object.keys(this.item.flags).includes('starwarsffg')) {
+              // the object is not properly set up yet; bail to let it finish
+              return;
+          }
           this.item.flags.starwarsffg.loaded = true;
 
-          const specializationTalents = data.system.talents;
+          const specializationTalents = data.data.talents;
```

Comment:
Is this one deliberately going from `system` to `data`?

Reply:
It's almost certainly an artifact of the weird merging I did. Good catch.

Reply:
And of course now that I've tested it... I can confirm that yes, it is intentionally going to `data.data`. It doesn't generate a warning and breaks with `system` so I think this should be left as-is.